schemas/develops.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "resource:/develops.json",
+  "title": "product",
+  "description": "Schema to describe a product at lobid.",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "type": {
+        "type": "string",
+        "const": "Role"
+      },
+      "roleName": {
+        "allOf": [
+          {
+            "$ref": "localizedString.json"
+          },
+          {
+            "enum": [
+              "NewProduct",
+              "Enhancement for"
+            ]
+          }
+        ]
+      },
+      "product": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "$ref": "resource:/id.json"
+          },
+          "name": {
+            "$ref": "resource:/name.json"
+          }
+        },
+        "anyOf": [
```

Comment:
In our KG, we should make `id`s a requirement for each resource except for qualified relationships (like `membership`). This means, we should just require `id` and remove the `anyOf` construct.